packages/auth-bundler/tests/bundler.spec.ts

```diff
@@ -55,7 +55,8 @@ describe('bundler.ts', function () {
 
       const content: BundleContent = {
         environment: Environment.PRODUCTION,
-        assets: [bundleContent.assets[1]],
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Remove this

Reply:
as this is eslint related mb it should wait for next PR?